docs/changelog/0.13.0.md

```diff
@@ -0,0 +1,272 @@
+---
+title: 0.13.0
+description: Changes in Typst 0.13.0
+---
+
+# Version 0.13.0
+
+## Highlights
+- There is now a distinction between [proper paragraphs]($par) and just
+  inline-level content. This is important for future work on accessibility and
+  means that [first line indent]($par.first-line-indent) can now be enabled for
+  all paragraph instead of just consecutive ones.
+- The [`outline`] has a better out-of-the-box look and is more customizable
+- The new [`curve`] function (that supersedes the `path` function) provides a
+  simpler and more flexible interface for creating Beziér curves
+- The `image` function now supports raw [pixel raster formats]($image.format)
+  for generating images from within Typst
+- Functions that accept [file paths]($syntax/#paths) now also accept raw
+  [bytes] instead, for full flexibility
+- WebAssembly [plugins]($plugin) are more flexible and automatically run
+  multi-threaded
+- Fixed a long-standing bug where single-letter strings in math (`[$"a"$]`)
+  would be displayed in italics
+- You can now specify which charset should be [covered]($text.font) by which
+  font family
+- The [`pdf.embed`] function lets you embed files into the exported PDF
```

Comment:
IIUC, this is *not* embedding PDF documents as part of your document (like with `include`), right? Maybe add a sentence making that clearer.